src/PlaywrightSharp/PointAndScroll.cs

```diff
@@ -4,9 +4,9 @@ namespace PlaywrightSharp
 {
     internal class PointAndScroll
     {
-        public decimal? ScrollX { get; set; }
+        public decimal ScrollX { get; set; }
 
-        public decimal? ScrollY { get; set; }
+        public decimal ScrollY { get; set; }

```

Comment:
Should we be able to scroll to x0y0?

Reply:
this class is only instantiated in a single place -> `ElementHandle.ViewportPointAndScrollAsync`